server/plugins/datasources/googlesheets/operations.ts

```diff
@@ -1,22 +1,53 @@
 import got from 'got';
 
-export async function readDataFromSheet(
+async function googleSheetGetValues(
```

Comment:
Will it be better to name it `getValuesFromSheet`?

Reply:
Agreed. Have updated to make it clearer.